packages/bvaughn-architecture-demo/components/console/MessageHoverButton.module.css

```diff
@@ -1,11 +1,13 @@
 .AddCommentButton,
 .ConsoleMessageHoverButton {
+  font-family: "Inter", sans-serif;
+  font-weight: 600;
```

Comment:
Ooo, hmm. Can we avoid using one-off `font-family` and `font-weigh` styles like this? Seems like we should have _one_ uniform font for components (well one for mono and one for not-mono).

Reply:
Not sure if you missed this comment but this still seems like a regression to me.

Reply:
Yeah, more design systems work 😅 would like to tackle this in a follow-up if that's ok.

Reply:
Can we just remove the font change for now then until we're ready to tackle the large font design stuff?

I know I'm being picky but I'd rather not leak one-off font families into the new CSS.

Reply:
No worries! I set to a sans-serif for now so it is at least closer to our other buttons and not monospace, but we can clean this up and do a proper pass on fonts.